sdk/face/ai-vision-face-rest/samples-dev/detect.ts

```diff
@@ -5,6 +5,9 @@ import createFaceClient, {
     isUnexpected,
 } from '@azure-rest/ai-vision-face';
 
+import * as dotenv from "dotenv";
```

Comment:
Can we just use `import "dotenv/config";` and not need the second line?

Reply:
@mpodwysocki Updated by your comments, please re-review.